tests/resources/preserve_module_path/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.24)
+project(MyApp CXX)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/modules)
+
+find_program(CCACHE ccache)
+find_package(OpenSSL)
+find_package(ZLIB)
+find_package(prometheus-cpp)
```

Comment:
This would be the thing to simplify, maybe adding a fake ``openssl-config.cmake`` in another folder and adding it to the CMAKE_PREFIX_PATH?

Reply:
Yea, these are the `find_package()` calls before the problem occurs. 
I'll do some digging in my environment to pinpoint exactly what it is.